jenkins/ci.suse.de/cloud-mkcloud7.yaml

```diff
@@ -30,6 +30,7 @@
         - 'cloud-mkcloud{version}-job-4nodes-linuxbridge-tempestfull-{arch}'
         - 'cloud-mkcloud{version}-job-upgrade-nondisruptive-{arch}'
         - 'cloud-mkcloud{version}-job-upgrade-nondisruptive-ha-{arch}'
+        - 'cloud-mkcloud{version}-job-upgrade-nondisruptive-ha-without-nodes-{arch}'
```

Comment:
super nit: `without-nodes-upgrade` ? Like is now seems to suggest that is not using nodes, instead of not upgrading the nodes